packages/office-addin-lint/src/defaults.ts

```diff
@@ -5,9 +5,21 @@ import * as usageData from "office-addin-usage-data";
 
 export const lintFiles = "src/**/*.{ts,tsx,js,jsx}";
 
+export enum ESLintExitCode {
+  Success = 0,
+  LintingError = 1,
```

Comment:
Based on https://eslint.org/docs/user-guide/command-line-interface#exit-codes

I think these values may be clearer. The first two are both "success" cases but I think it's better to focus on the fact that it indicates whether or not there were lint errors reported. For the third case, the command itself failed.

```
NoLintErrors = 0,
HasLintError = 1,
CommandFailed = 2
```